File: Programs/SolutionST2/climbStairs.cpp
int climbStairs(int n){
  // WRITE ALL YOUR CODE HERE, IN THIS FUNCTION
  int prev = 1;
  int sum = 2;
  if (n == 1)
    return 1;

  for (int i = 2; i < n; i++) {
    int temp = sum;
    sum = prev + sum;
    prev = temp;
  }


  return sum;
}

File: Programs/SolutionST2/climbStairs.py
def climbStairs(n):
    # WRITE ALL YOUR CODE HERE, IN THIS FUNCTION
    prev = 1
    sum = 2
    if (n == 1):
        return 1

    for i in range(2, n):
        temp = sum
        sum = prev + sum
        prev = temp

    return sum
    pass
